configure.ac

```diff
@@ -302,6 +302,17 @@ AC_ARG_WITH([cc-image],
     ]
 )
 
+AC_ARG_ENABLE([autogopath],
+	AS_HELP_STRING([--enable-autogopath], [enable autogopath @<:@default=false@:>@]),
```

Comment:
Could you add a bit more detail here? I think the description should mention `$GOPATH` (and ideally why you'd want to use this option if you already have a `$GOPATH` set).